core/trino-spi/src/main/java/io/trino/spi/block/DecimalAggregationAccumulatorBlock.java

```diff
@@ -29,28 +28,28 @@
 import static io.trino.spi.block.BlockUtil.compactArray;
 import static io.trino.spi.block.BlockUtil.countUsedPositions;
 
-public class FixedBinaryBlock
-        implements Block, FixedBinaryBlockOutput
+public class DecimalAggregationAccumulatorBlock
+        implements Block
 {
-    private static final int INSTANCE_SIZE = ClassLayout.parseClass(FixedBinaryBlock.class).instanceSize();
+    private static final int INSTANCE_SIZE = ClassLayout.parseClass(DecimalAggregationAccumulatorBlock.class).instanceSize();
 
     private final int positionOffset;
     private final int positionCount;
     @Nullable
     private final boolean[] valueIsNull;
     private final long[] values;
+    private final long[] overflow;
 
     private final long sizeInBytes;
     private final long retainedSizeInBytes;
-    private final int positionWidthBytes;
```

Comment:
I don't think these change. I would treat `overflow` similarly as we treat `valueIsNull`